modules/installation-launching-installer.adoc

```diff
@@ -212,7 +217,23 @@ and compute nodes.
 .. Select the base domain to deploy the cluster to. All DNS records will be
 sub-domains of this base and will also include the cluster name.
 endif::osp[]
+ifdef::vsphere[]
+.. Select *vsphere* as the platform to target.
+.. Specify the name of your vCenter instance.
+.. Specify the user name and password for the vCenter account that has the required permissions to create the cluster.
++
+The installation program connects to your vCenter instance.
+.. Select the datacenter in your vCenter instance to connect to.
+.. Select the default vCenter datastore to use.
```

Comment:
After the step of selecting the default vCenter datastore, missed one option of " Select the default vCenter cluster"